packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -146,7 +146,7 @@ async function clientConfig(env) {
 				'process.env.ADD_SW': env.sw,
 				'process.env.PRERENDER': env.prerender,
 			}),
-			new PushManifestPlugin(),
+			new PushManifestPlugin(env.isProd),
```

Comment:
Moving this into the prod config seems like a better move at first glance, but `renderHTMLPLugin` depends on it being ran first so we need to keep it here.